includes/class-content-toc.php

```diff
@@ -233,8 +233,8 @@ public function filter_content( $post_content, $shortcode_atts ) {
 			$list_html = sprintf(
 				'<%1$s%2$s>%3$s</%1$s>',
 				esc_attr( $this->settings['list_tag'] ),
-				$this->tag_class( $this->settings['list_class'] ),
-				$items_html
+				esc_attr( $this->tag_class( $this->settings['list_class'] ) ),
```

Comment:
The `tag_class()` method returns a complete class attribute (eg. ` class="foo"`), so passing it through `esc_attr()` means it won't work and will end up as ` class=&quot;foo&quot;`. Same applies to the changes below too.